src/tools/simple-kernel-timer-json/kp_kernel_info.h

```diff
@@ -6,7 +6,26 @@
 #include <sys/time.h>
 #include <cstring>
 
+#if defined(__GXX_ABI_VERSION)
+#define HAVE_GCC_ABI_DEMANGLE
+#endif
 
+#if defined(HAVE_GCC_ABI_DEMANGLE)
+#include <cxxabi.h>
+#endif // HAVE_GCC_ABI_DEMANGLE
+
+char* demangleName(char* kernelName)
```

Comment:
The definition should really go in in a source file, not in a header file.  Otherwise, you could get multiple definition link errors, etc.